application/routers/entity.py

```diff
@@ -56,13 +56,21 @@ def _get_geojson(data: List[EntityModel]) -> Dict[str, Union[str, List[GeoJSON]]
     return {"type": "FeatureCollection", "features": features}
 
 
-def _get_entity_json(data: List[EntityModel], include: Optional[Set] = None):
+def _get_entity_json(
```

Comment:
This should have some new unit tests for what you're doing in tests/unit/routers/test_entity.py